packages/block-editor/src/components/block-tools/index.js

```diff
@@ -95,6 +95,17 @@ export default function BlockTools( {
 		} else if (
 			isMatch( 'core/block-editor/delete-multi-selection', event )
 		) {
+			/**
+			 * Check if the target element is a text area, input or
+			 * event.defaultPrevented and return early. In all these
+			 * cases backspace could be handled elsewhere.
+			 */
+			if (
+				[ 'INPUT', 'TEXTAREA' ].includes( event.target.nodeName ) ||
+				event.isDefaultPrevented()
```

Comment:
I think this should be `event.defaultPrevented`.

Reply:
Thanks 😄

Reply:
I had tried it in debugger and saw the function and used it. I didn't know `jQuery` enhances the event object with that function. That was interesting 😄